### ISSUE TITLE
unable to build on Raspberry Pi...src/io.c:129: undefined reference to g_simple_io_stream_new

### ISSUE THREAD

**Issue by salfter** [uzbl/uzbl#271]
Maybe there's an undeclared dependency somewhere, but I'm trying to build the latest version as the one in the Raspbian repo doesn't recognize the show_status option (I want to hide the status bar for a kiosk installation).  I installed libwebkit-dev as described in http://www.uzbl.org/wiki/ubuntu, but the build of uzbl-core fails with an "undefined reference to g_simple_io_stream_new." libglib2.0-dev was already pulled in by something else.

(This is on a clean install of the latest Raspbian, brought up-to-date with the usual apt-get update/dist-upgrade/autoremove.)

**Comment by mathstuf** [comment]
Seems that `g_simple_io_stream_new` is [new in 2.44](https://developer.gnome.org/gio/unstable/GSimpleIOStream.html#g-simple-io-stream-new).

**Comment by keis** [comment]
you could try the [0.9.0 version](https://github.com/uzbl/uzbl/releases/tag/v0.9.0) which does not use iostream but should still be much more recent than what raspbian carries.

It looks like 2.48 is in jessie-backports (not sure if that carries over to raspbian though) so I'm not to worried about just bumping the required version, WDYT @mathstuf ?

**Comment by mathstuf** [comment]
The new I/O stuff is nice; not really interested in ifdeffery again either, so bumping is fine with me.

**Comment by salfter** [comment]
Tried again with tags/v0.9.0 instead of master...this time, it built without error.  

Another site recommended creating a kiosk configuration with the following:

```
set show_status=0
set geometry=maximized
```

and then starting uzbl with ```-c /home/pi/uzbl.conf```, or something to that effect.  However, the status bar was still showing up.  With some experimentation, I settled on this:

```uzbl-core -g maximized http://invalid.example.tld/```

This runs full-screen, without the status bar.  The window manager is set to not draw titlebars.  So far, uzbl appears to handle scaling of large images better than epiphany-browser, which was sometimes taking seconds to resize a large image.
